test/functional/tests/cache_ops/test_multistream_seq_cutoff.py

```diff
@@ -0,0 +1,244 @@
+                      f"seq-cutoff threshold value minus one 4k block."):
+        kib_between_streams = 100
+        max_range_offset = \
+            int(streams_number * (threshold.get_value(Unit.KibiByte) + kib_between_streams))
+        range_step = int(threshold.get_value(Unit.KibiByte)) + kib_between_streams
+        offsets = [o for o in range(0, max_range_offset, range_step)]
+        core_statistics_before = core.get_statistics()
+
+        for i in TestRun.iteration(range(0, len(offsets))):
+            TestRun.LOGGER.info(f"Statistics before I/O:\n{core_statistics_before}")
+
+            offset = Size(offsets[i], Unit.KibiByte)
+            run_dd(core.system_path, count=int(threshold.get_value(Unit.Blocks4096) - 1),
+                   seek=int(offset.get_value(Unit.Blocks4096)))
+
+            core_statistics_after = core.get_statistics()
+            check_statistics(core_statistics_before,
+                             core_statistics_after,
+                             expected_pt=0,
+                             expected_writes_to_cache=threshold - Size(1, Unit.Blocks4096))
+            core_statistics_before = core_statistics_after
+
+    with TestRun.step("Write random number of 4k block requests to each stream and check if all "
+                      "writes were sent in pass-through mode."):
+        core_statistics_before = core.get_statistics()
+        random.shuffle(offsets)
+
+        for i in TestRun.iteration(range(0, len(offsets))):
+            TestRun.LOGGER.info(f"Statistics before second I/O:\n{core_statistics_before}")
+            additional_4k_blocks_writes = random.randint(1, kib_between_streams / 4)
+            offset = Size(offsets[i], Unit.KibiByte)
+            run_dd(
+                core.system_path, count=additional_4k_blocks_writes,
+                seek=int(offset.get_value(Unit.Blocks4096)
+                         + threshold.get_value(Unit.Blocks4096) - 1))
+
+            core_statistics_after = core.get_statistics()
+            check_statistics(core_statistics_before,
+                             core_statistics_after,
+                             expected_pt=additional_4k_blocks_writes,
+                             expected_writes_to_cache=Size.zero())
+            core_statistics_before = core_statistics_after
+
+
+def check_statistics(stats_before, stats_after, expected_pt, expected_writes_to_cache):
+    TestRun.LOGGER.info(f"Statistics after I/O:\n{stats_after}")
+    writes_to_cache_before = stats_before.block_stats.cache.writes
+    writes_to_cache_after = stats_after.block_stats.cache.writes
+    pt_writes_before = stats_before.request_stats.pass_through_writes
+    pt_writes_after = stats_after.request_stats.pass_through_writes
+
+    actual_pt = pt_writes_after - pt_writes_before
+    actual_writes_to_cache = writes_to_cache_after - writes_to_cache_before
+    if actual_pt != expected_pt:
+        TestRun.LOGGER.error(f"Expected pass-through writes: 0\n"
```

Comment:
expected_pt don't have to be always 0, right? If we can specify that argument please print out passed value instead of harcoded 0 :)

Reply:
You're right ;) fixed